includes/pages/admin_free.php

```diff
@@ -58,8 +58,8 @@ function admin_free() {
     $free_users_table[] = array(
         'name' => User_Nick_render($usr),
         'shift_state' => User_shift_state_render($usr),
-        'dect' => $usr['DECT'],
-        'jabber' => $usr['jabber'],
+        'handy' => $usr['Handy'],
```

Comment:
As the column in the database is named "Handy" (the db naming is inconsistent as ...), I just keeped using the same name